content/en/docs/setup/install/helm/index.md

```diff
@@ -166,6 +166,15 @@ installed above.
     $ kubectl delete namespace istio-system
     {{< /text >}}
 
+## Uninstall revision tags
+
+If you decide to rollback to the old control plane, instead of completing the canary upgrade,
+you can uninstall the canary revision using `istioctl x uninstall --revision=canary` or
+`helm template istiod manifests/charts/istio-control/istio-discovery -s templates/revision-tags.yaml --set revisionTags={prod-stable} --set revision=canary -n istio-system | kubectl delete -f -`.
+
+However, in this case you must first reinstall the gateway(s) for the previous revision manually,
+because the uninstall command will not automatically revert the previously in-place upgraded ones.
```

Comment:
```suggestion
If you decide to continue using the old control plane, instead of completing the upgrade,
you can uninstall the newer revision using `istioctl x uninstall --revision=canary` or
`helm template istiod manifests/charts/istio-control/istio-discovery -s templates/revision-tags.yaml --set revisionTags={prod-stable} --set revision=canary -n istio-system | kubectl delete -f -`.

However, in this case you must first reinstall the gateway(s) for the previous revision manually,
because the uninstall command will not automatically revert the previously in-place upgraded ones.
```

Reply:
I'm confused about these commands and why should we mention `istioctl x uninstall` command here?